src/pages/steps/LocationStep.tsx

```diff
@@ -512,6 +511,11 @@ const LocationStep = ({
             field?.value as OfferFormData['location'];
 
           const isDefaultZip = municipality?.zip === '0000';
+          const isBrusselsZip =
+            municipality?.zip &&
+            Number(municipality.zip) >= 1000 &&
+            Number(municipality.zip) <= 1210;
```

Comment:
I'm a bit late to review, but I wonder if we should also check the selected country here to be safe? I see that right now we also support The Netherlands and Germany, and unless I'm mistaken their zip codes should never be in this range. So technically it's not necessary, but it might be safe in case we add more countries at some point and forget about this check? Unless it's a lot of trouble.

Reply:
Good point, I would be safer. 
I can make a follow-up PR